enhancements/cluster-logging/hypershift-audit-logs.md

```diff
@@ -211,59 +161,42 @@ Downgrading: removes new features but otherwise works as expected.
 
 ### Version Skew Strategy
 
-The CLO's installed on management and hosted clusters may be different versions.
+Data and control plane may have different versions of CLO installed. Points of contact:
 
-The CLO on the _management_ cluster forwards
-- All logs required by the provider, configured by `ClusterLogForwarderTemplate` on _management_ cluster
-- API-audit logs _only_ for the customer, configured by `HypershiftLogForwarder` on _hosted_ cluster
+- HTTP input-to-output used to forward audit logs.
+- API audit policy configuration.
 
-The CLO on the _hosted_ cluster forwards
-- All non-API-audit logs, configured by `ClusterLogForwarder` on the _hosted_ cluster.
+As long as configuration for these two items remains compatible, the CLO versions can vary.
+These are protected by normal backwards-compatibility rules between releases.
 
-If the hosted/managent CLO versions are different, the customer can
-- Create `ClusterLogForwarder` based on the _hosted_ CLO version.
-- Create `HypershiftLogForwarder` based on the _management_ CLO version.
-
-This might be confusing but there is a clear separation of resources associated with each version.
-- There's no situation where the customer is blocked from upgrading their hosted CLO.
-- A customer could be blocked from sending API-audit logs (only) to a new type of log store if the SD CLO is old.
-
-To manage version skew we will:
-- minimize API change via normal API compatibility practices.
-- identify supported version range, code the HLO to correct or reject configuration version mismatches in that range.
+It is important to verify and test this compatibility with CLO and HCP releases,
+so we can advertise the correct supported version ranges.
```

Comment:
Where does QE responsibility for this live?  And specifically in the context of Red Hat Managed HCP?